src/redux/actions/pipeline/runGem2s.js

```diff
@@ -46,7 +46,8 @@ const runGem2s = (experimentId) => async (dispatch, getState) => {
       payload: {
         experimentId,
         experimentName: experiments[experimentId].name,
-        projectUuid: experiments[experimentId].projectUuid,
+        projectId,
```

Comment:
Why not keep the previou way of passing the projectId payload? We already getting name from that anyways, right?

Reply:
`projetUuid` is stored as `projectId` in `experiments` table in dynamoDB. That is also the key used in the `experimentInfo` reducer. unfortunately, the way it is stored in `info` is under teh key `projectUuid`. I think this is a major problem. We have several items that means the same things but under different naming conventions : 
- samples : sampleId(s), sampleUuid(s), sample(s)
- project : projectId, projectUuid

I vaguely remember there is a ticket about this? I think it's quite a refactor involving all parts of the platform.

```
const updateExperimentInfo = produce((draft, action) => {
  const {
    experimentId,
    experimentName,
    projectId,
    sampleIds,
  } = action.payload;

  draft.info.experimentId = experimentId;
  draft.info.experimentName = experimentName;
  draft.info.projectUuid = projectId;
  draft.info.sampleIds = sampleIds;

  // Experiment id was updated so processing config requires reloading
  draft.processing.meta.loading = true;
}, initialState);
```